driver/src/main/java/org/neo4j/driver/Session.java

```diff
@@ -87,67 +88,169 @@
      * @param work the {@link TransactionWork} to be applied to a new read transaction.
      * @param <T> the return type of the given unit of work.
      * @return a result as returned by the given unit of work.
+     * @deprecated superseded by {@link #executeRead(TransactionCallback)}.
      */
+    @Deprecated
     <T> T readTransaction( TransactionWork<T> work );
 
     /**
-     * Execute a unit of work in a managed {@link AccessMode#READ read} transaction
-     * with the specified {@link TransactionConfig configuration}.
+     * Execute a unit of work as a single managed transaction with {@link AccessMode#READ read} access mode and retry behaviour.
```

Comment:
```suggestion
     * Execute a unit of work as a single, managed transaction with {@link AccessMode#READ read} access mode and retry behaviour. The transaction allows for one or more statements to be run.
```

Something similar on the other variants (the info about one or more statements).

Reply:
Thanks, updated.